### PR TITLE
Prevent Next hydration error 

### LINKED ISSUE #1042: Prevent Next hydration error
image

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Bug Fix

## Related issues

Resolve #1042

### CODE REVIEW

File: src/harmony/Badge/Badge.tsx [hunk 1]
```diff
@@ -77,21 +77,21 @@ export const Badge = forwardRef(
                 className={cn(s.Badge, className, view ? viewMap[view] : '', sizeMap[size])}
             >
                 {nullable(iconLeft, (icon) => (
-                    <span className={cn(s.BadgeIcon, { [s.BadgeIconLeft]: text })}>{icon}</span>
+                    <div className={cn(s.BadgeIcon, { [s.BadgeIconLeft]: text })}>{icon}</div>
```
Comment (tsumo) [inline src/harmony/Badge/Badge.tsx:80]: Why did it cause hydration issues?

Comment (asabotovich) [reply src/harmony/Badge/Badge.tsx:80]: Next doesn't like `div` inside `span`
image

**Review by tsumo — APPROVED** [review]

**Pull request merged**